ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -111,6 +111,8 @@ public class PeerDiscoveryController {
   private final PeerTable peerTable;
   private final Cache<Bytes, DiscoveryPeer> bondingPeers =
       CacheBuilder.newBuilder().maximumSize(50).expireAfterWrite(10, TimeUnit.MINUTES).build();
+  private final Cache<Bytes, Packet> cachedEnrRequests =
+      CacheBuilder.newBuilder().maximumSize(25).expireAfterWrite(1, TimeUnit.MINUTES).build();
```

Comment:
Instead of hardcoding size to 25, maybe set it to the configured max peers?

Reply:
yes clearly

Reply:
after reflection I will not link it to this value because if we put a very low max peers I do not want the cache to be too low. I will rather link to the maximum number of bonding peers. which seems a bit more logical